src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+const {
+  getExpenses,
+  createExpense,
+  getExpense,
+  deleteExpense,
+  updateExpense,
+} = require('../services/expenses.serveces');
+
+const getAllExpenses = (req, res) => {
+  const {
+    userId,
+    categories,
+    from,
+    to,
+  } = req.body;
+
+  if (!userId || !categories || !from || !to) {
+    res.sendStatus(400);
```

Comment:
It's better to also provide a description of what went wrong, not just the status code.